PATHFIND/GetVehicleNodeProperties.md

```diff
@@ -8,17 +8,29 @@ ns: PATHFIND
 BOOL GET_VEHICLE_NODE_PROPERTIES(float x, float y, float z, int* density, int* flags);
 ```
 
-```
+## Description
 Gets the density and flags of the closest node to the specified position.  
 Density is a value between 0 and 15, indicating how busy the road is.  
-Flags is a bit field.  
-```
 
 ## Parameters
-* **x**: 
-* **y**: 
-* **z**: 
-* **density**: 
-* **flags**: 
+* **x**: x position of search
+* **y**: y position of search
+* **z**: z position of search
+* **density**: returned density of traffic ranging from 0 - 15
+* **flags**: returned flags of this node (see below)
+
+### Flags
+* **0x1**:    Offroad
+* **0x2**:    Unknown. Potentially a target for wander tasks as it follows game camera.
```

Comment:
This has been dynamically tagged as ahead, potentially situated on or close to the road the player is currently on.